cpu/cpu.go

```diff
@@ -330,7 +339,7 @@ func getStats(path string, stats map[string]map[string]interface{}, prevMetricsS
 			}
 			cpuID = allCPU //change CPU identifier for aggregation metrics
 		} else {
-			cpuID = cpuID[len(cpuID)-1:] //get the last character
+			cpuID = cpuID[cpuStrLength:] //get number from CPU indentifier, for example if CPU identifier is cpu42 then 42 is get
```

Comment:
How do You think about TrimPrefix "cpu" from string cpuID? I would like to avoid declaring additional variable `cpuStrLenght`  if it can be done without it.